### PR TITLE
sync headers fetch progress

### PR DESCRIPTION
**Pull request by ReevesAk**
**Fixes** issue #95 

The pr was created to determine the actual number of headers fetched in the sync operation and use it to ascertain fetch progress.

### CODE REVIEW

**Comment by RogueElement** [comment]
Add a description of why you're correcting this and how this correction is valid.

**Comment by itswisdomagain** [comment]
> Add a description of why you're correcting this and how this correction is valid.

Specifically, mention the issue that this PR is meant to solve and give a brief description of the correction you've effected.

File: syncnotification.go [hunk 1]
```diff
@@ -108,7 +108,8 @@ func (mw *MultiWallet) fetchHeadersProgress(lastFetchedHeaderHeight int32, lastF
 
 	headersLeftToFetch := mw.estimateBlockHeadersCountAfter(lastFetchedHeaderTime)
 	totalHeadersToFetch := lastFetchedHeaderHeight + headersLeftToFetch
-	headersFetchProgress := float64(lastFetchedHeaderHeight) / float64(totalHeadersToFetch)
+	headersFetchedSoFar := mw.syncData.beginFetchTimeStamp
```
Comment (itswisdomagain) [inline syncnotification.go]: How's the total number of headers fetched so far equal to the begin fetch timestamp?

Comment (ReevesAk) [reply syncnotification.go]: It does not, file has some changes.

File: syncnotification.go [hunk 2]
```diff
@@ -108,7 +108,8 @@ func (mw *MultiWallet) fetchHeadersProgress(lastFetchedHeaderHeight int32, lastF
 
 	headersLeftToFetch := mw.estimateBlockHeadersCountAfter(lastFetchedHeaderTime)
 	totalHeadersToFetch := lastFetchedHeaderHeight + headersLeftToFetch
-	headersFetchProgress := float64(lastFetchedHeaderHeight) / float64(totalHeadersToFetch)
+	headersFetchedSoFar := mw.syncData.activeSyncData.startHeaderHeight
+	headersFetchProgress := float64(headersFetchedSoFar) / float64(totalHeadersToFetch)
```
Comment (C-ollins) [inline syncnotification.go:112]: This does not address the issue and does not seem valid.

**Comment by C-ollins** [comment]
Closed via #143

**Pull request closed without merging**